vhd/packer/sysprep.ps1

```diff
@@ -0,0 +1,38 @@
+# Stop and remove Azure Agents to enable use in Azure Stack
+# If deployed in azure vm the agents will be added to the vms at deployment time
```

Comment:
Do you plan to stop DiagTrack?

Reply:
We only want to stop DiagTrack in the pause image which is being done in: https://github.com/kubernetes/kubernetes/pull/95950